docs/cpp/enable-logging-cpp.md

```diff
@@ -28,3 +27,38 @@ To turn on full logging for an issue report, follow these steps:
 1. Select the "C/C++" option in the log filter selector:
 
    ![Log filter selector](images/cpp/log-filter-selector.png)
+
+## Enable logging for the debugger
```

Comment:
Can you change this to say "for the debug adapter"

Reply:
Are there any other places where the "debugger" should be referred to as "debug adapter"?

Reply:
those were the only two where i thought it would be good to have the distinction.